package.json

```diff
@@ -41,20 +41,38 @@
     "uuid": "^7.0.2"
   },
   "scripts": {
-    "server": "node server.js",
-    "start": "react-scripts start",
+    "build-storybook": "build-storybook -s public",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "docs": "jsdoc -c jsdoc.conf.json",
     "eject": "react-scripts eject",
+    "format": "npm run lint:fix && npm run prettier:fix",
+    "lint:fix": "eslint ./src --fix --ext .js,.jsx,.ts,.tsx",
+    "lint": "eslint ./src --ext .js,.jsx,.ts,.tsx",
+    "prettier:check": "prettier --check ./src",
+    "prettier:fix": "prettier --write ./src",
+    "server": "node server.js",
+    "start": "react-scripts start",
     "storybook": "start-storybook -p 9009 -s public",
-    "build-storybook": "build-storybook -s public",
-    "docs": "jsdoc -c jsdoc.conf.json",
-    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
-    "prettier:check": "prettier --check .",
-    "prettier:fix": "prettier --write ."
+    "test": "react-scripts test"
```

Comment:
Just added ./src to the lint and format rules otherwise we were trying to lint and fix node_modules too. (another alternative now that i think about it is to just add in an .eslintignore)

And added in the format script + sorting the rest of the scripts object